cdc/processor.go

```diff
@@ -773,6 +782,8 @@ func (p *processor) addTable(ctx context.Context, tableID int64, replicaInfo *mo
 	globalcheckpointTs := atomic.LoadUint64(&p.globalcheckpointTs)
 
 	if replicaInfo.StartTs < globalcheckpointTs {
+		// use Warn instead of Panic in case that p.globalcheckpointTs has not been initialized.
+		// The cdc_state_checker will catch a real inconsistency in integration tests.
```

Comment:
If `globalcheckpointTs` is larger than 0 (initialized already), and `replicaInfo.StartTs < globalcheckpointTs`, then it is a bug, right?

Reply:
Yes